src/toil/batchSystems/abstractBatchSystem.py

```diff
@@ -47,6 +48,12 @@ def __init__(self, config, maxCores, maxMemory, maxDisk):
         """
         :type dict[str,str]
         """
+        self.workerCleanupInfo = {
```

Comment:
Yes, if the named tuple is defined and setup here, It can't be easily pickled and unpickled outside this scope.
http://stackoverflow.com/questions/16377215/how-to-pickle-a-namedtuple-instance-correctly

easier to keep it a dict that is easily pickle-able